E01-Geometry/Benas/Index.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require "src/Shape.php";
+require "src/shapes/Circle.php";
+require "src/shapes/Rectangle.php";
+require "src/shapes/Rhombus.php";
+
+// Circle
+$circle = new Circle(2);
+echo $circle->calculateArea($circle->radius, $circle->pi)." ";
```

Comment:
If `$circle` already contains `pi`, why do you need to pass it via method arguments?
Circle already knows PI.